m2m-transfers/V3/REPWRITERSPECS/BNO.INTERACTIVE.PRO

```diff
@@ -250,12 +258,16 @@ PROCEDURE NTADDCROSSACCOUNTS
      NTPREFACCESSID=PREFERENCE ACCESS:ID
      FOR ACCOUNT PREFERENCE ACCESS:ACCOUNTNUMBER
       DO
-       IF NTPREFACCESSIDTYPE=2 OR NTPREFACCESSIDTYPE=0 THEN
+       BNODEBUGMSG=FORMAT("** NTSLTYPE:99:",NTSLTYPE)
+       CALL BNODEBUGMSGADD
+       IF (NTPREFACCESSIDTYPE=2 OR NTPREFACCESSIDTYPE=0) AND
+          (NTCROSSACCTSLTYPE=NTSLTYPESHARE OR
```

Comment:
Hi Tom, Is this fix actually a Bug in BNO.INTERACTIVE.PRO that is an improvement for any other PO's that may use that functionality in that include?

Reply:
@jordan-keenan , yes, the issue is resolved by a change in the ".PRO"